routes/psnr.js

```diff
@@ -2,16 +2,29 @@ const PsnrResult = require('../models/psnrResult');
 
 module.exports = function(app) {
     app.post('/psnrResult', (req, res) => {
-        const { buildNum, buildUrl, psnrValue } = req.body;
+        const {
+            buildNum,
+            buildUrl,
+            psnr,
+            numFrozenFrames,
+            numSkippedFrames,
+            totalFrames
+        } = req.body;
 
-        console.log(`received psnr ${psnrValue}`
+        console.log(`received psnr ${psnr},`
+            + ` numFrozenFrames ${numFrozenFrames},`
+            + ` numSkippedFrames ${numSkippedFrames},`
+            + ` totalFrames ${totalFrames},`
             + ` for build num ${buildNum}`
             + ` url ${buildUrl}`);
 
         PsnrResult.create({
             buildNum,
             buildUrl,
-            psnrValue
+            psnr,
+            numFrozenFrames,
+            numSkippedFrames,
```

Comment:
Are route/frameSkip and models/frameSkipResult still used?

Reply:
nope, forgot to get rid of those.